types/project.d.ts

```diff
@@ -72,7 +72,7 @@ export interface Project {
   url?: string;
   contacts?: {
     name: string;
-    email?: string;
+    email: string;
```

Comment:
In the old showcase, the email was not mandatory, because the owner often disappeared after some time. Then the system used the prof email from the lab, IIRC. Or is this something different here?

Reply:
OK, now I see: it's only the email of the contact. Then it makes sense to have this mandatory.